packages/main/src/plugin/documentation/documentation-service.ts

```diff
@@ -0,0 +1,247 @@
+      },
+    ];
+
+    for (const config of parseConfigs) {
+      try {
+        for (const item of config.data) {
+          if (item.name && item.url) {
+            documentation.push({
+              id: `${config.category}-${this.generateId(item.name)}`,
+              title: item.name,
+              description: `${config.category}: ${item.name}`,
+              url: item.url,
+              category: config.category,
+            });
+          }
+        }
+      } catch (error) {
+        console.error(config.errorMessage, error);
+      }
+    }
+
+    // Remove duplicates and add core pages
+    const corePages = this.getCoreDocumentationPages();
+    return [...corePages, ...this.removeDuplicates(documentation)];
+  }
+
+  private getCoreDocumentationPages(): DocumentationInfo[] {
+    return [
+      {
+        id: 'docs-intro',
+        title: 'Introduction & Getting Started',
+        description: 'Learn the basics of Podman Desktop',
+        url: 'https://podman-desktop.io/docs/intro',
+        category: 'Documentation',
+      },
+      {
+        id: 'tutorial-index',
+        title: 'Tutorials & Guides',
+        description: 'Step-by-step tutorials for common tasks',
+        url: 'https://podman-desktop.io/tutorial',
+        category: 'Tutorial',
+      },
+    ];
+  }
+
+  private getFallbackDocumentation(): DocumentationInfo[] {
+    return [
+      {
+        id: 'docs-intro',
+        title: 'Introduction & Getting Started',
+        description: 'Learn the basics of Podman Desktop',
+        url: 'https://podman-desktop.io/docs/intro',
+        category: 'Documentation',
+      },
+      {
+        id: 'tutorial-index',
+        title: 'Tutorials & Guides',
+        description: 'Step-by-step tutorials for common tasks',
+        url: 'https://podman-desktop.io/tutorial',
+        category: 'Tutorial',
+      },
+      {
+        id: 'docs-containers',
+        title: 'Containers Documentation',
+        description: 'Working with containers, images, and pods',
+        url: 'https://podman-desktop.io/docs/containers',
+        category: 'Documentation',
+      },
+      {
+        id: 'docs-kubernetes',
+        title: 'Kubernetes Documentation',
+        description: 'Deploy and manage Kubernetes applications',
+        url: 'https://podman-desktop.io/docs/kubernetes',
+        category: 'Documentation',
+      },
+      {
+        id: 'docs-extensions',
+        title: 'Extensions Development',
+        description: 'Create and develop extensions',
+        url: 'https://podman-desktop.io/docs/extensions/developing',
+        category: 'Documentation',
+      },
+      {
+        id: 'docs-troubleshooting',
+        title: 'Troubleshooting Guide',
+        description: 'Solve common issues and problems',
+        url: 'https://podman-desktop.io/docs/troubleshooting',
+        category: 'Documentation',
+      },
+    ];
+  }
+
+  private removeDuplicates(items: DocumentationInfo[]): DocumentationInfo[] {
```

Comment:
Why is this needed ? I would rather have the docusaurus script ensuring that all links are unique